packs/promtail/metadata.hcl

```diff
@@ -0,0 +1,11 @@
+app {
+  url    = "https://grafana.com/docs/promtail/latest/clients/promtail"
+  author = "Ricky Grassmuck"
```

Comment:
I believe this is expected to be the author of the application, rather than the pack.